src/printing.jl

```diff
@@ -213,7 +213,7 @@ function print_sourcecode(io::IO, code::String, line::Integer, defline::Integer)
 
     for textline in code
         printstyled(io,
-            string(lineno, " "^(stoplinelength-length(lineno)+1));
```

Comment:
This didn't make sense at all to me. `lineno` is an integer so `length` of it is always zero. And why should we indent a different amount depending on the current linenumber?